packages/vaadin-date-picker/test/basic.test.js

```diff
@@ -207,6 +203,11 @@ describe('basic features', () => {
     expect(datepicker.hasAttribute('has-value')).to.be.true;
   });
 
+  it('should format the input value', () => {
+    const datepicker = fixtureSync('<vaadin-date-picker value="2000-01-01"></vaadin-date-picker>');
+    expect(datepicker.inputElement.value).to.equal('1/1/2000');
+  });
```

Comment:
Let's move this test to a separate suite e.g. `initial value attribute` - something like this:
https://github.com/vaadin/web-components/blob/60b1d69664dc1d5070c5daa98b8729d193abca7d/packages/vaadin-date-time-picker/test/basic.test.js#L228-L241

Reply:
Done